design-documents/graph-ql/coverage/customer/Wishlist.graphqls

```diff
@@ -1,19 +1,19 @@
 type Mutation {
     createWishlist(name: String!): ID # Multiple wishlists Commerce functionality
-    removeWishlist(id: ID!): Boolean # Commerce fucntionality - in Opens Source we assume customer always has one wishlist
+    removeWishlist(uid: ID!): Boolean # Commerce fucntionality - in Opens Source we assume customer always has one wishlist
     addProductsToWishlist(wishlistId: ID!, wishlistItems: [WishlistItemInput!]!): AddProductsToWishlistOutput
     removeProductsFromWishlist(wishlistId: ID!, wishlistItemsIds: [ID!]!): RemoveProductsFromWishlistOutput
     updateProductsInWishlist(wishlistId: ID!, wishlistItems: [WishlistItemUpdateInput!]!): UpdateProductsInWishlistOutput
 }
 
 type Customer {
     wishlist: Wishlist! @deprecated(reason: "Use `Customer.wishlists` or `Customer.wishlist_v2") 
-    wishlist_v2(id: ID!): Wishlist # This query will be added in the ce
+    wishlist_v2(uid: ID!): Wishlist # This query will be added in the ce
```

Comment:
We should just deprecate wishlist field and use new field wishlists, which will always have one item in Open Source.

Reply:
@melnikovi I see, you have updated the schema file, Is there a need to do any additional changes to the schema file?